Documentation/contributing/testing/ci.rst

```diff
@@ -141,6 +141,33 @@ Once you change the image versions locally, create a branch named
 It is important that you use that branch name so the VM images are cached into
 packet.net before the branch is merged.
 
+
+Testing matrix
+^^^^^^^^^^^^^^
+
+We are currently testing following kernel - k8s version pairs in our CI:
+
++--------------------+------------------+
+| Kubernetes version | Kernel version   |
++====================+==================+
+| Vagrant k8s clusters per PR           |
++--------------------+------------------+
+| 1.11               | 5.x.x (net-next) |
```

Comment:
small nit: I'd probably just specify latest `bpf-next` above.

Reply:
Should we update this every time we update the vagrant image to new kernel? Also, what is the latest `bpf-next`?

Reply:
Ideally the docs should keep in sync with reality, yeah. IMO the best way is if we can actually point to some repo somewhere as a single source-of-truth for docs and for actual testing. But these docs are better than nothing.

"Latest `bpf-next`" is basically saying that the `bpf-next` tree was pulled at the timestamp indicated in the latest build here: 
https://jenkins.cilium.io/job/Vagrant-Master-Boxes-Packer-Build-net-next/

It's a moving target, currently tracking what will become Linux v5.7. If we can state in the docs that it's a "latest" kernel development tree then we don't have to go and update the docs every time we bump the tree. It won't be out-of-date/wrong. And at least in the past few weeks, we have been changing this a few times. The idea is that it's constantly moving to track any potential regressions upstream.

Trade-off of course is that it doesn't explicitly communicate which actual version. For most datapath developers, I think we likely understand what `bpf-next` / `net-next` means, and folks can always ask on Slack.

Reply:
(technically if someone wanted to trace all the way then they need to look at the timestamp of the above build, then look at the `packer-ci-build` tree at that commit, for this line:
https://github.com/cilium/packer-ci-build/blob/master/provision/ubuntu/kernel-next-bpftool.sh#L11 ..)

Currently it's pointing at the Linux `bpf` tree, which is pretty close to `bpf-next` (it just has an extra fix which I needed for some Cilium development). We'll swap back to `bpf-next` when `bpf`/`bpf-next` trees merge again this week so I think that stating "latest `bpf-next`" in the docs would typically be the most accurate description.